tools/test/audio/test_utils/test_run.m

```diff
@@ -92,14 +92,16 @@
 	fprintf(fh, 'XTRUN=\n');
 end
 
-% Override defaults in comp_run.sh
-fprintf(fh, 'VALGRIND=false\n');
 fclose(fh);
 
 arg = sprintf('-t %s', fn_config);
 rcmd = sprintf('%s %s', ex, arg);
 fprintf('Running ''%s''...\n', rcmd);
-system(rcmd);
+ret = system(rcmd);
+if ret
+	error('''%s'' returned status %d\n', rcmd, ret);
+end
+
```

Comment:
At line number 58, `fprintf(fh, '#!/bin/sh\n', test.comp);` Isn't the output format strange?

Reply:
Good catch: it does look like a bug but how is it related to this PR?